Base/usr/share/man/man5/font.md

```diff
@@ -1,17 +1,42 @@
 ## Name
 
-font - Bitmap Font File format
+font - Bitmap Font File format (.font)
 
 ## Synopsis
 
-The .font file format stores bitmap fonts in SerenityOS's own binary format.
+Font files contain bitmap definitions of fonts (Gfx::BitmapFont).
 
 ## Description
 
-These files contain bitmap definitions of fonts, either varying-width or fixed-width.
+Bitmap fonts can be either varying-width or fixed-width.
 
-The first four bytes contain the filemagic: `!Fnt` (0x21466e74).
+There is a list of features:
```

Comment:
Please remove this and try to work the below list into prose, it's easier to read this way.